tests/phpunit-admin.xml

```diff
@@ -0,0 +1,15 @@
+<phpunit bootstrap="bootstrap-admin.php">
+    <php>
+      <server name="kernel.environment" value="test" />
```

Comment:
~~Why do you need to declare this server value ? isn't ``putenv`` and this declaration strictly equivalents ?